### ISSUE TITLE
Variable Names

### ISSUE THREAD

**Issue by Zayelion** [Fluorohydride/ygopro-pre-script#74]
I wanted to start an initiative to rename the various single or two letter variable name conventions used by YGOPro. I dont feel it is good practice to write code this way with few exception, like iterators, rare errors, or mathematics. These are objects that map to functions inside `ygopro-core` and obscure a new developer from understanding the overall API.

We would start by identifying the variable names that need to be renamed. Pick names for them, then as convention going forward change the names. I'd also start a sister project in tandem where files are renamed one by one in the ygopro-script repo. Its not an issue of man-power, I have enough volunteers ready for this.

I wanted to get input first.

**Comment by DiabladeZat** [comment]
Yeah, I think that we should rename them. When I was learning, one major point of confusion was these variables.

**Comment by SharkYGOPRO** [comment]
Hi, You can create a script of the Supreme King Servant Dragon Dark Rebellion anime effect?

**Comment by EerieCode** [comment]
Why are you asking this in a completely unrelated issue?

**Comment by Ejeffers1239** [comment]
Yeah, Seems like a good idea, I pretty much just started scripting and these conventional variables have confused me. You sort it out eventually, but it is just not a good time
